guidelines/taxation.md

```diff
@@ -0,0 +1,38 @@
+# Taxation
+
+## Legal environments and Tax codes
```

Comment:
```suggestion
## Legal environments, tax environments, and tax codes in Mews
```

Reply:
Why T as capital? :)

`## Legal environments and tax codes`